cms/templates/import.html

```diff
@@ -48,11 +72,23 @@ <h3 class="title">
 
           <input type="file" name="course-data" class="file-input" />
 
-          <input type="submit" value="${_('Replace my course with the one above')}" class="submit-button" />
+          <input type="submit"
+          %if library:
+              value="${_('Replace my library with the one above')}"
```

Comment:
This one is interesting, because I would imagine for the most part this kind of change should impact both libraries and courses. (Which is fine for us to change here, but I think it might make sense to try and keep them both in sync, regardless of which direction we pick.) The suggested change makes sense to me.